### ISSUE TITLE
aoi: review and revise contrast of sidenav menu items (accessibility)

### ISSUE THREAD

**Issue by sjtrimble** [angular/angular#17218]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Contrast may be too low to meet minimum accessibility requirements. Reported by @wardbell 

**Expected behavior**
Contrast should meet minimum accessibility requirements.

**Minimal reproduction of the problem with instructions**

- Go to https://aio-staging.firebaseapp.com/docs
- Open sidenav drawer to view font color of sidenav menu items

**Comment by gkalpak** [comment]
Update:
- The current color for top-level sidenav items (`#444444` on `#fafafa`) passes both WCAG AA and WCAG AAA.
- The current color for nested sidenav items (`#6e6e6e` on `#fafafa`) passes WCAG AA but not WCAG AAA.

**Comment by sjtrimble** [comment]
This is SUPER old, we should be able to close.

**Comment by gkalpak** [comment]
But we still fail WCAG AAA for nested sidenav items (which is probably the least of our a11y issues, but let's keep this around for tracking anyway) :grin:

**Comment by sjtrimble** [comment]
Is that a requirement? I thought AA was the requirement, AAA was 'nice to have'? 

Ok.

**Comment by gkalpak** [comment]
TBH, I don't know if we have made a decision on what our requirements are 🤷‍♂

**Comment by JeanMeche** [comment]
This is no longer an issue on both light & dark theme.
